src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -379,6 +379,8 @@ public IEnumerable<WorkspaceModel> Workspaces
         internal static string DefaultPythonEngine { get; private set; }
 
         internal static DynamoUtilities.DynamoFeatureFlagsManager FeatureFlags { get; private set; }
+
+        internal bool ForceAutomaticWithoutRun { get; set; } = false;
```

Comment:
I think this new property makes more sense to have on the `RunSettings` class, which can be accessed from `HomeWorkspaceModel`. Once we set it there, its value should be accessible from almost anywhere. It feels out of place on `DynamoModel`. This way I hope we can also avoid creating a new command to open files in `automaticWithoutRun` mode.

Reply:
@aparajit-pratap this property was moved to RunSettings.
commit: https://github.com/DynamoDS/Dynamo/pull/12947/commits/9e9530081f1fe1a44784ee009ee220886e4dc7c3
Also I removed the added command since is not needed:
https://github.com/DynamoDS/Dynamo/pull/12947/commits/807b63447b80954a4e0dca4f0feb354efa435203